test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/gainorlossafterreliefs/GainOrLossAfterReliefsControllerSpec.scala

```diff
@@ -228,10 +237,10 @@ class GainOrLossAfterReliefsControllerSpec
         mockUUIDGenerator
       )
 
-      "display the page" when {
+      "display the page when calculator disabled" when {
```

Comment:
the controller isn't concerned with whether or not the calculator is disabled - it's concerned with whether the eligibility service says the user is eligible for calculation or not